test_case/ERA5TimeSeriesDataset.py

```diff
@@ -266,24 +265,28 @@ def seqzarr_pipeline():
     Pipeline to load Zarr stores via a DALI External Source operator.
     """
     # Zarr source
-    source = SeqZarrSource()
+    source = SeqZarrSource(batch_size=16)
+
+    def index_generator(batch_info: dali.types.BatchInfo) -> np.ndarray:
+        return np.array([batch_info.iteration])
 
-    # Update length of dataset
-    # self.total_length: int = len(source) // self.batch_size
+    indexes = dali.fn.external_source(
+        source=index_generator, dtype=dali.types.INT64, batch_info=True
```

Comment:
The last parameter here should be `batch=True` instead of `batch_info=True` right?

Reply:
So [`batch=True`](https://docs.nvidia.com/deeplearning/dali/archives/dali_1_47_0/user-guide/operations/nvidia.dali.fn.external_source.html#nvidia.dali.fn.external_source.params.batch) should already be set by default. You're correct though that we don't need `batch_info`, have removed that in commit 05ddfa39db7793954066554b4ac148712def657c